modules/dists/SparseBlockDist.chpl

```diff
@@ -66,10 +66,12 @@ class SparseBlockDom: BaseSparseDom {
   param rank: int;
   type idxType;
   param stridable: bool = false;  // TODO: remove default value eventually
-  const dist: Block(rank, idxType);
+  type sparseLayoutType = DefaultDist;
```

Comment:
Are you sure we need to set the default here?
Does just
     type sparseLayoutType ;
work ?